### DISCUSSION TITLE
Get all possible projections?

### DISCUSSION THREAD

**Discussion by ajdapretnar** [pyproj4/pyproj#896] [Q&A] [answered]
Is it possible to extract all supported projections with full info, i.e. `("World Geodetic System", "WGS 84", "EPSG: 4326")`?

Use case: We are building a widget for a GUI application where we'd like to use pyproj for transforming from one system to another and would like to offer all supported projections to the user (with full names, not codes).

**Comment by snowman2** [comment, marked as answer]
This should help: https://pyproj4.github.io/pyproj/stable/api/database.html#pyproj-database-query-crs-info

**Reply by snowman2** [reply]
Also see #661

**Reply by ajdapretnar** [reply]
Thanks a million! I was traversing the code for ages, but couldn't fine the `query_crs_info`. It is what I need.

And thanks for the second link. I am already using `tranformer.itransform`, which works very well.

**Reply by snowman2** [reply]
Glad to hear that this was resolved. The global context is also something worth looking into.
